dist/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "root": true,
+
+  "extends": "../.eslintrc-browser.json",
+  "rules": {
+	// That is okay for the built version
+	"no-multiple-empty-lines": "off"
+  }
+}
```

Comment:
I don't think this file is used right now as we don't lint `dist/sizzle.js`. Either that needs to be added to Gruntfile (as it is in jQuery) together with the modifications in `.eslintignore` that I listed above or this file needs to be removed.

Reply:
I have updated `eslint` task in the `Gruntfile` and now this file should be used